__tests__/commands/workspaces.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import {BufferReporter} from '../../src/reporters/index.js';
+import {run as workspace} from '../../src/cli/commands/workspaces.js';
+import * as reporters from '../../src/reporters/index.js';
+import Config from '../../src/config.js';
+import path from 'path';
+
+const fixturesLoc = path.join(__dirname, '..', 'fixtures', 'workspace');
+
+async function runWorkspaces(
+  flags: Object,
+  args: Array<string>,
+  name: string,
+  checkSteps?: ?(config: Config, reporter: BufferReporter) => ?Promise<void>,
+): Promise<void> {
+  const cwd = path.join(fixturesLoc, name);
+  const reporter = new reporters.BufferReporter({stdout: null, stdin: null, isSilent: true});
+
+  try {
+    const config = await Config.create({cwd}, reporter);
+
+    await workspace(config, reporter, flags, args);
+
+    if (checkSteps) {
+      await checkSteps(config, reporter);
+    }
+  } catch (err) {
+    throw new Error(`${err && err.stack}`);
```

Comment:
Why are we doing this instead of rethrowing the error or just removing the whole `try`/`catch` block?

Reply:
Good question. I took this code from the `workspace` test, but I can't remember why I'm doing this.